proto/evmos/auctions/v1/genesis.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+syntax = "proto3";
+package evmos.auctions.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/evmos/evmos/v18/x/auctions/types";
+
+// GenesisState defines the auctions module's genesis state.
+message GenesisState {
+  // params defines all the parameters of the module.
+  Params params = 1 [(gogoproto.nullable) = false];
```

Comment:
In the genesis we should also include the auction (bid and deposit state objects)